pre-stage0/seeds.txt

```diff
@@ -0,0 +1,7 @@
+NodeID - 9eb35adc39052493bf704fe53f95f784d55108fc
```

Comment:
Add if the seed is for heimdall or bor, maybe create 2 files or make this file `.md`